torch/csrc/jit/codegen/cuda/iter_visitor.cpp

```diff
@@ -97,11 +133,61 @@ void IterVisitor::traverseFrom(
   }
 }
 
+// This is a simplified version of traverseFrom
+void IterVisitor::traverseFrom2(
+    Fusion* const fusion,
+    const std::vector<Val*>& from,
+    bool traverseAllPaths,
+    bool respectComputeAt) {
+  FusionGuard fg(fusion);
+  std::unordered_set<Statement*> visited;
+  stmt_stack.clear();
+  if (!from.empty())
```

Comment:
I'd recommend using { ... } even for single statements, any thoughts?

Reply:
I can't agree more, but this is also just copied from `traverseFrom`.

Reply:
I'm lazy those keys are hard to hit :-P, just kidding, that's fine, I'll stop leaving them off.

Reply:
thanks! (even LLVM, which used to be hardcore on not using braces for single statements, is not reconsidering it)